packages/odyssey-react-mui/src/labs/SideNav.tsx

```diff
@@ -0,0 +1,572 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  memo,
+  useMemo,
+  useState,
+  MouseEvent,
+  ReactElement,
+  ReactNode,
+} from "react";
+
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "../OdysseyDesignTokensContext";
+
+import { NavAccordion } from "./NavAccordion";
+
+import { Status, statusSeverityValues } from "../Status";
+
+import { Box } from "../Box";
+import type { HtmlProps } from "../HtmlProps";
+import styled from "@emotion/styled";
+import { Heading6 } from "../Typography";
+import { CollapseLeftIcon, ExpandLeftIcon } from "../icons.generated";
+import { Link } from "../Link";
+
+export type SideNavItem = {
+  id: string;
+  label: string;
+  target?: string;
```

Comment:
Alphabetize these, please!

(Though I like having the required ones at the top, so maybe just alphabetize the optional props)